gst/nnstreamer/nnstreamer_conf.c

```diff
@@ -468,6 +474,9 @@ nnsconf_get_custom_value_string (const gchar * group, const gchar * key)
     if (NULL == value && conf.conffile) {
       g_autoptr (GKeyFile) key_file = g_key_file_new ();
 
+      g_assert (key_file != NULL);
+      g_assert (conf.conffile != NULL);
```

Comment:
A `NULL` check for `conf.conffile` is already performed at 474. I think we can remove one.

Reply:
Will update this.